src/apps/explorer/pages/AppData/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useCallback, useState, useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+import { faCode, faListUl } from '@fortawesome/free-solid-svg-icons'
+import { useQuery } from 'hooks/useQuery'
+import FormPage from './FormPage'
+import DecodePage from './DecodePage'
+import TabIcon from 'components/common/Tabs/TabIcon'
+import { TabItemInterface } from 'components/common/Tabs/Tabs'
+
+import { ContentCard as Content, Title } from 'apps/explorer/pages/styled'
+import { StyledExplorerTabs, Wrapper } from './styled'
+
+enum TabView {
+  FORM = 1,
+  DECODE,
+}
+
+const DEFAULT_TAB = TabView[1]
+
+function useQueryViewParams(): { tab: string } {
+  const query = useQuery()
+  return { tab: query.get('tab')?.toUpperCase() || DEFAULT_TAB } // if URL param empty will be used DEFAULT
+}
+
+const tabItems = (): TabItemInterface[] => {
+  return [
+    {
+      id: TabView.FORM,
+      tab: <TabIcon title="Form" iconFontName={faListUl} />,
```

Comment:
I'm not sold on the `Form` title.
I'd be inclined to go with the opposite operation to the other tab: `encode`